conan/api/model/list.py

```diff
@@ -306,7 +306,7 @@ def refs(self):
                 result[recipe] = rrev_dict
         return result
 
-    def items(self) -> Iterable[tuple[RecipeReference, dict[PkgReference, dict]]]:
+    def items(self) -> Iterable[Tuple[RecipeReference, Dict[PkgReference, dict]]]:
```

Comment:
```suggestion
    def items(self) -> Iterable[Tuple[RecipeReference, Dict[PkgReference, Dict]]]:
```
What about the inner dict?

Reply:
We also use ``dict`` as type in other places like ``deps_cpp_info_dirs: dict``

Reply:
The issue here is using dict with explicit arguments, but using `dict` for "returned object" is perfectly valid